stats/metric_values.go

```diff
@@ -0,0 +1,98 @@
+package stats
+
+import (
+	"context"
+	"sync"
+
+	"go.opentelemetry.io/otel/attribute"
+	api "go.opentelemetry.io/otel/metric"
+)
+
+type metricAttributes struct {
+	baseAttribCount		int
+	MetricName			string
+	Attribs 			[]attribute.KeyValue
+}
+
+func newMetricAttribs(metricName string, baseAttribs []attribute.KeyValue) *metricAttributes {
```

Comment:
can you expand "Attribs" to "Attributes" throughout for consistency?

Reply:
Updated